app_management/utils/handlers.rb

```diff
@@ -51,7 +51,7 @@ def get_handler(name)
     def validate(enabled_handlers)
       valid = Set.new
       invalid = Set.new
-      enabled_handlers.each do |enabled_handler|
+      enabled_handlers.sort.each do |enabled_handler|
```

Comment:
If it was just so the order was always the same for the new tests then it could be sorted in the test after calling this method.